src/super_gradients/recipes/training_hyperparams/coco2017_yolox_train_params.yaml

```diff
@@ -1,7 +1,7 @@
 defaults:
   - default_train_params
 
-max_epochs: 300
+max_epochs: 1
```

Comment:
Don't forget to revert back when done testing)

Reply:
Sure, removed